### PR TITLE
Bootstrap service

### PR DESCRIPTION
**Pull request by yurivin**
Quorum returned by /iroha/config/accounts/{project}/{env} endpoint does not consider number of Iroha peers #426

### CODE REVIEW

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/controller/IrohaController.kt [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
 
-    @GetMapping("/config/accounts/{project}/{env}")
-    fun getNeededAccounts(@PathVariable("project") project: String, @PathVariable("env") env: String): ResponseEntity<List<AccountPrototype>> {
+    @GetMapping("/config/accounts/{project}/{env}/{peersCount}")
+    fun getNeededAccounts(
+        @PathVariable("project") project: String,
+        @PathVariable("env") env: String,
+        @PathVariable("peersCount") peersCount: Int
+    ): ResponseEntity<NeededAccountsResponse> {
+        if (peersCount == null || peersCount == 0) {
```
Comment (dolgopolovwork) [inline bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/controller/IrohaController.kt]: But `peersCount` cannot be nullable. Or am I wrong?

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/dto/Model.kt [hunk 2]
```diff
@@ -49,11 +48,24 @@ data class IrohaAccount(val title: String, val domain: String, val keys: HashSet
     }
 }
 
-data class Peer(@NotNull val peerKey: String = "", @NotNull val hostPort: String = "localhost:10001")
-data class AccountPublicInfo(@NotNull val pubKeys: List<String> = emptyList(), @NotNull val domainId: String? = null, @NotNull val accountName: String? = null)
+data class Peer(
+    @NotNull val peerKey: String = "",
+    @NotNull val hostPort: String = "localhost:10001",
```
Comment (dolgopolovwork) [inline bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/dto/Model.kt]: Why `10001`?

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestGenesisFactory.kt [hunk 3]
```diff
@@ -14,7 +14,15 @@ import java.util.*
 class D3TestGenesisFactory : GenesisInterface {
 
     private val zeroPubKey = "0000000000000000000000000000000000000000000000000000000000000000"
-    override fun getAccountsNeeded(): List<AccountPrototype> = D3TestContext.d3neededAccounts
+    override fun getAccountsForConfiguration(peersCount: Int): List<AccountPrototype> {
+        val activeAccounts = D3TestContext.d3neededAccounts.filter { !it.passive }
+        activeAccounts.stream().forEach {
```
Comment (dolgopolovwork) [inline bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestGenesisFactory.kt]: Can we remove `stream()`? Is it going to work without this call?

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/dto/Model.kt (hunk 2 above)
Comment (yurivin) [reply bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/dto/Model.kt]: I found this value as default in block

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestGenesisFactory.kt (hunk 3 above)
Comment (yurivin) [reply bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestGenesisFactory.kt]: fixed

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/controller/IrohaController.kt (hunk 1 above)
Comment (yurivin) [reply bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/controller/IrohaController.kt]: fixed

**Pull request merged**